### PR TITLE
[DRAFT] appleseed: 2.0.5-beta -> 2.1.0-beta

### PR DESCRIPTION
**Pull request by onny**
###### Description of changes

fix build and update package

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by winterqt** [comment]
@ofborg eval

File: pkgs/tools/graphics/appleseed/default.nix [hunk 1]
```diff
@@ -39,46 +76,36 @@ in stdenv.mkDerivation rec {
...
+  postInstall = ''
+    chmod a+x $out/bin/*
+    wrapProgram $out/bin/appleseed.studio --set PYTHONHOME ${python2}
+  '';
+
   meta = with lib; {
-    description = "Open source, physically-based global illumination rendering engine";
+    description = ''
+      Open source, physically-based global illumination rendering engine
+    '';
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/appleseed/default.nix:105]:
```suggestion
    description = "Open source, physically-based global illumination rendering engine";
```

File: pkgs/tools/graphics/appleseed/default.nix [hunk 2]
```diff
@@ -39,46 +76,36 @@ in stdenv.mkDerivation rec {
...
+    "-DWITH_PYTHON=ON"
+    "-DWITH_DISNEY_MATERIAL=ON"
+    "-DUSE_SSE=ON"
+    "-DUSE_SSE42=ON"
   ];
 
+  # Work around a bug in the CMake build:
+  postInstall = ''
+    chmod a+x $out/bin/*
+    wrapProgram $out/bin/appleseed.studio --set PYTHONHOME ${python2}
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/appleseed/default.nix:99]: Do we really need to use python2?

File: pkgs/tools/graphics/appleseed/default.nix [hunk 3]
```diff
@@ -1,35 +1,72 @@
...
+, seexpr
+, wrapQtAppsHook }:
+
+let
+
+  boost_static = boost170.override {
+    enableStatic = true;
+    enablePython = true;
+    python = python2;
+  };
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/appleseed/default.nix:29]: We should move this to top-level, too.

**Comment by NickCao** [comment]
appleseed has been removed from nixpkgs.

**Pull request closed without merging**